smoke_tests/happypath/resume.feature

```diff
@@ -0,0 +1,44 @@
+Feature: Resume to continue existing application
+
+  Background:
+    Given I show my environment
+    Then I visit "/"
+    And I should see "What do you want to do?"
+    When I click the "Appeal against a tax decision" link
+    Then I should see "Appeal against a tax decision"
+    And I click the "Continue" link
+    Then I should see "What is your appeal about?"
+    Given I choose "Income Tax"
+    Then I should see "Did you appeal the original decision to HMRC?"
```

Comment:
I think we can get rid of this as it's in another test
'Then I should see "Did you appeal the original decision to HMRC?"'